lms/djangoapps/course_home_api/outline/v1/views.py

```diff
@@ -247,3 +285,28 @@ def dismiss_welcome_message(request):
         return Response({'message': _('Welcome message successfully dismissed.')})
     except Exception:
         raise UnableToDismissWelcomeMessage
+
+
+@api_view(['POST'])
+@authentication_classes((JwtAuthentication, SessionAuthenticationAllowInactiveUser,))
+@permission_classes((IsAuthenticated,))
+def save_course_goal(request):
```

Comment:
We must have already had an endpoint for this, yeah? What's the difference between the two? (Format of success/fail messages?)